tests/InArrayValidatorTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace ipl\Tests\Validator;
+
+use ipl\I18n\NoopTranslator;
+use ipl\I18n\StaticTranslator;
+use ipl\Validator\InArrayValidator;
+
+class InArrayValidatorTest extends TestCase
+{
+    public function testInArrayValidatorWithValidValue()
+    {
+        StaticTranslator::$instance = new NoopTranslator();
+
+        $validator = new InArrayValidator([
+            'haystack' => ['test', 'foo', 'bar', 5, [], 0.008]
+        ]);
+
+        $this->assertTrue($validator->isValid('foo'), 'foo exists in the given haystack');
```

Comment:
I'm quite sure the message is used if the assertion fails. So, `foo not found in ..` is correct.

Reply:
This seems to apply to all of your messages, in all test cases. Please adjust all of them.